app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+            text-decoration: none;
+        }
+        &:focus{
+            color: $text_secondary;
+            background-color: $yellow_secondary;
+            border-bottom: 4px solid $support_focus;
+            text-decoration: none!important;
+            outline: none;
+        }
+    }
+// ---------------------------------------------------------------------------------
+
+// -- Buttons ----------------------------------------------------------------------
+.button{
+    box-sizing: border-box;
+    font-family: 'Roobert-SemiBold', sans-serif;
+    &:active, &:focus, &:hover{
+        outline: none!important;
+    }
+    &.button--primary,
+    &.button--small{
+        background: $magenta_primary;
+        color: $white!important;
+        text-decoration: none;
+        font-size: 16px;
+        border: 3px solid $magenta_primary;
+        border-radius: $border_radius;
+        box-shadow: 0px 3px 0px rgba(0, 11, 59, 0.25);
+        padding: 8px 16px;  
+        &:hover{
+            background: $magenta_dark;
+            border: 3px solid $magenta_dark;
+        }
+        &:focus{
+            background-color: $yellow_secondary;
+            border: 3px solid $support_focus;
+            color: $text_secondary;
+        }
+        &:active{
+            background: $magenta_darker;
+            color: $white;
+            border: 3px solid $yellow_secondary;
+            box-shadow: 0px 3px 0px $white, inset 0px 5px 2px rgba(0, 11, 59, 0.25);
+        }
+    }
+    &.button--small{
+        padding: 11px 16px 0 16px;
+        &:focus{
+            color: $text_secondary!important;
+        }
+    }
+
+    &.button--secondary {
+        background: $white;
+        color: $magenta_primary;
+        font-size: 16px;
+        border: 1px solid $magenta_primary;
+        border-radius: $border_radius;
+        box-shadow: 0px 3px 0px rgba(0, 11, 59, 0.25);
+        padding: 12px 16px;
+
+        &:hover{
+            //background: $light_grey_2;
+            border: 1px solid $magenta_dark;
+        }
+        &:focus{
+            background-color: $yellow_secondary;
+            border: 3px solid $support_focus;
+            color: $text_secondary;
+        }
+        &:active{
+            font-weight: 600;
+            background: $light_grey_2;
+            color: $text_secondary;
+            border: 3px solid $support_focus;
+            box-shadow: 0px 3px 0px $white, inset 0px 5px 2px rgba(0, 11, 59, 0.25);
+        }
+    }
+}
+
+
+// ---------------------------------------------------------------------------------
+
+
+// -- Form elements ----------------------------------------------------------------
+label{
+    font-family: 'Roobert-Regular', sans-serif;
+    font-size: 16px;
+    color: $text_secondary;
+}
+input[type=text],
+input[type=number],
+input[type=email],
+input[type=password], 
+textarea{
+    width: 285px;
+    height: 40px;
+    font-family: 'Roobert-Regular', sans-serif;
```

Comment:
Can we define the fallback font within the `@font-face` declaration to avoid the need for duplication?